src/game.rs

```diff
@@ -20,9 +20,11 @@ pub const POINT_LIGHT_COLOR: Vector3<f32> = Vector3::new(0.93126976, 0.7402633,
 // pub const LIGHT_COLOR_C: Vector3<f32> =
 //     Vector3::new(from_srgb(0.631), from_srgb(0.565), from_srgb(0.627));
 
-pub const COLLISION_GROUP_BASE: u32 = 1;
-pub const COLLISION_GROUP_PLAYER_SHOOTABLE: u32 = COLLISION_GROUP_BASE << 1;
-pub const COLLISION_GROUP_PLAYER_UNSHOOTABLE: u32 = COLLISION_GROUP_PLAYER_SHOOTABLE << 1;
+#[allow(dead_code)]
```

Comment:
I think it makes sense to just delete these unused ones since they don't even do anything right now, maybe just do:

```rust
pub const COLLISION_GROUP_PLAYER_UNSHOOTABLE: Group = Group::GROUP_1;
```

I tested it and it seems to work fine. I'm glad you've updated all these deps, their APIs seem to have some nice improvements like this enum instead of the crazy bitshifting :D

Reply:
Nice!